README.md

```diff
@@ -182,17 +182,14 @@ which lives in `Data.Functor.Variant`. `VariantF` is just like `Variant`,
 except it's indexed by things of kind `Type -> Type`.
 
 ```purescript
-someFoo :: forall v. VariantF (foo :: FProxy Maybe | v) Int
-someFoo = inj (SProxy :: SProxy "foo") (Just 42)
+someFoo :: forall v. VariantF (foo :: Proxy Maybe | v) Int
+someFoo = inj (Proxy :: Proxy "foo") (Just 42)
 
-someBar :: forall v. VariantF (bar :: FProxy (Tuple String) | v) Int
```

Comment:
This proxy needs to be removed per my changes (and the similar cases in the following lines)

Reply:
Ah, right. Thanks. That's been fixed.